source/features/notification.d.ts

```diff
@@ -1,5 +1,8 @@
 export interface Notification {
-	participants: { username: string | null; avatar: string }[];
+	participants: Array<{
+		username: string | null;
+		avatar: string;
+	}>;
```

Comment:
Why `Array`?

Reply:
Also, worth dropping that `null`?

Reply:
It's more readable to only use `type[]` shorthand for simple types.

https://github.com/xojs/eslint-config-xo-typescript/commit/7b1f11891a4c2a3282792fcb139c7e134fe7f7dd

> Also, worth dropping that null?

Yes, but it's not really related to dependency updates.

Reply:
Code compiles without the `null`, so I've removed it.